pkg/resource/s_ecr_et/delta.go

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Something weird happened with this file directory name? I'm seeing `pkg/resource/s_ecr_et/delta.go`?

Reply:
no idea  why the generator make this,  I try `v0.26.0` and `v0.26.1`,  need to check generator

Reply:
run a simple test in generator,   `crd.Names.Snake` has issue with `secret`
```
    model_secretsmanager_test.go:39: 
                Error Trace:    /Users/y.chu/golang/src/github.com/aws-controllers-k8s/code-generator/pkg/model/model_secretsmanager_test.go:39
                Error:          Not equal: 
                                expected: "secret"
                                actual  : "s_ecr_et"
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -secret
                                +s_ecr_et
                Test:           TestSecretsManager_Secret
```

Reply:
it uses `ecr` to make snake.
```go
initialisms = []initialismTranslator{
                ......
      		{"Ecr", "ECR", "ecr", nil},
                .....
}
```